.template/addons/bootstrap/application.js.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+insert_into_file 'app/javascript/packs/application.js', after: %r{import 'core-js/stable'.+\n} do
```

Comment:
@rafayet-monon  is this needed to follow here https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RegexpLiteral ?

Reply:
I think it's as per the rule as there's a `/` in the regex. What do you think? @noor-ahmed

Reply:
@rafayet-monon got it 🙌